### ISSUE TITLE
Nested SVG elements unsupported

### ISSUE THREAD

**Issue by Carreau** [jupyter/notebook#982]
@brainstorm opened ipython/ipython#8996

The plot shows up exactly as advertised in the README.md of the official project **when opened directly via a browser** (as in File->Open File->image.svg):

https://github.com/CSB-IG/pyveplot

But it shows only partially when trying to render it with IPython's Display SVG:

https://gist.github.com/brainstorm/25f69d7a735f6e0127ff

**Comment by parleur** [comment]
... I am not sure this is a notebook error as the following code produce the desired result ...
```.python
from IPython.display import HTML
nested_svg="""
<svg xmlns="http://www.w3.org/2000/svg">
  <svg x="0">
    <rect x="10" y="10" height="80" width="80" style="fill: #0000ff"/>
  </svg>
  <svg x="100">
    <rect x="10" y="10" height="80" width="80" style="fill: #00cc00"/>
  </svg>
</svg>"""
HTML(nested_svg)
```

**Comment by parleur** [comment]
Solved in ipython repo [#9214](https://github.com/ipython/ipython/pull/9214)
